compute/instances.go

```diff
@@ -963,8 +974,7 @@ func (api *_Instance) toNative() (*Instance, error) {
 // submit an API call to the vmapi machine endpoint
 func (mcns *InstanceCNS) toTags(m map[string]interface{}) {
 	if mcns.Disable != nil {
-		s := fmt.Sprintf("%t", mcns.Disable)
-		m[CNSTagDisable] = &s
+		m[CNSTagDisable] = fmt.Sprintf("%t", *mcns.Disable)
```

Comment:
Fair enough, I think I'm going to remove the pointers here as well. I'm not sure why they're used by the `InstanceCNS` struct values.